piplapis/search.py

```diff
@@ -90,7 +92,9 @@ class SearchAPIRequest(object):
     """
 
     HEADERS = {'User-Agent': 'piplapis/python/%s' % piplapis.__version__}
-    BASE_URL = '{}://api.pipl.com/search/?'
+    DEFAULT_API_VERSION = 5
```

Comment:
pay attention that here the version is integer but in piplapis.__version__ is is string. it may cause problems down the road

Reply:
one is the SDK version and the other is the API version